scripts/globals/items/resolution_ring.lua

```diff
@@ -13,7 +13,7 @@ local item_object = {}
 
 item_object.onItemCheck = function(target)
     local result = 0
-    if (target:hasStatusEffect(xi.effect.DEDICATION) == true) then
+    if target:hasStatusEffect(xi.effect.DEDICATION) == true then
```

Comment:
Just one other note, this line can be simplified to `if target:hasStatusEffect(xi.effect.DEDICATION) then`

Reply:
Ok - I will go through some of the other bands and fix these and put into a new PR.

Reply:
That sounds great^^
But please, edit this concrete one in this PR, so we minimize possible conflicts

Reply:
done sir